recipes/transformer-engine/build_te.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -euxo pipefail
+
+mkdir -p $PREFIX/etc/conda/activate.d
+cp $RECIPE_DIR/transformer-engine-env.sh $PREFIX/etc/conda/activate.d/
+
+cat > $RECIPE_DIR/gcc_shim <<"EOF"
+#!/bin/sh
+exec $GCC -I$PREFIX/include "$@"
+EOF
+
+chmod +x $RECIPE_DIR/gcc_shim
+export CC="$RECIPE_DIR/gcc_shim"
+
+# Create a symlink to the nvvm directory where transformer-engine's Cmake expects it.
+ln -s $PREFIX/nvvm $PREFIX/targets/x86_64-linux/nvvm
+cp -n $BUILD_PREFIX/targets/x86_64-linux/include/*.h $PREFIX/targets/x86_64-linux/include
+cp $PREFIX/include/cudnn*.h $PREFIX/targets/x86_64-linux/include
+
+echo "Installing transformer-engine"
+${PYTHON} -m pip install .
```

Comment:
Logs show that the pip package version metadata contains some kind of hash.

e.g.  transformer_engine-1.13.0+6a31486c93

Is this intentional? Or is there an override variable you can set.



https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1235681&view=logs&j=df9408f7-e795-500f-143d-aa7735a0ed8b&t=47addc80-ccbf-566b-ee3a-66cdb8436937&l=2088

Reply:
Good catch; I think I can override this with NVTE_NO_LOCAL_VERSION. Will do
https://github.com/NVIDIA/TransformerEngine/blob/d33525249d08ee6b2c80fded1b326248a2a1cb77/build_tools/te_version.py#L15